docs/reference/query-dsl/boosting-query.asciidoc

```diff
@@ -1,36 +1,76 @@
 [[query-dsl-boosting-query]]
 === Boosting Query
 
-The `boosting` query can be used to effectively demote results that
-match a given query. Unlike the "NOT" clause in bool query, this still
-selects documents that contain undesirable terms, but reduces their
-overall score.
+Returns documents matching a `positive` query while reducing the
+<<query-filter-context, relevance score>> of documents that also match a
+`negative` query.
 
-It accepts a `positive` query and a `negative` query.
-Only documents that match the `positive` query will be included
-in the results list, but documents that also match the `negative` query
-will be downgraded by multiplying the original `_score` of the document
-with the `negative_boost`.
+You can use the `boosting` query to demote certain documents without
+excluding them from the search results.
+
+[[boosting-query-ex-request]]
+==== Example request
 
 [source,js]
---------------------------------------------------
+----
 GET /_search
 {
     "query": {
         "boosting" : {
             "positive" : {
                 "term" : {
-                    "field1" : "value1"
+                    "text" : "apple"
                 }
             },
             "negative" : {
                  "term" : {
-                     "field2" : "value2"
+                     "text" : "pie tart fruit crumble tree"
```

Comment:
nice one!

Reply:
Shamelessly borrowed from https://www.elastic.co/guide/en/elasticsearch/guide/current/not-quite-not.html